README.md

```diff
@@ -176,6 +177,18 @@ http server framework. Some new events were added related to fastify setup.
     assembled via all the previous events has been added to `fastify` so
     that it can be served on `bedrock-server.ready`.
 
+## License
```

Comment:
bedrock and bedrock-express are some of our oldest repos.  Most open source repos we've done since then have had a copy & pasted support section above the final license section.  While we're fixing these up, should do the same, something like the below:

```md
## Commercial Support

Commercial support for this library is available upon request from
Digital Bazaar: support@digitalbazaar.com.

## License
...
```

Reply:
@davidlehn sorry, yeah, we've had commenting and review happening in too many places, so this didn't get updated yet.

Here's what we did upstream in `digitalbazaar/bedrock`'s readme:
https://github.com/digitalbazaar/bedrock?tab=readme-ov-file#license

We could go back to a separate section, but my fear is that it'd be overlooked vs. putting it here with the other licensing text.

Reply:
Fixed in [3213403](https://github.com/digitalbazaar/bedrock-express/pull/56/commits/3213403b61fa9d195c321ab65113270a7ab62a87).

If you want it done differently, let's discuss it elsewhere as a follow-up to keep this change moving.